### PR TITLE
XSS Vulnerability fix

### PR DESCRIPTION
**Pull request by DrunkSunGod**
## Previous Behavior

XSS Vulnerabilities arise when you set the html directly. This can be exploited by changing label names to reveal sensitive information like document.cookie
Consider label name "{alert(document.cookie)}> for example. Instead of displaying the tooltip, this would reveal the cookie when hovering over the label

## New Behavior

Setting the text content instead helps prevent XSS attacks. Changes have been tested locally and work as expected

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by AtishayMsft** [comment]
@DrunkSunGod could you provide a reproduction of the XSS possibility in the tooltip code for which you have raised a fix.

**Comment by DrunkSunGod** [comment]
https://github.com/user-attachments/assets/37677609-818a-45c3-a1d5-d293386257c9

@AtishayMsft FYI

**Comment by DrunkSunGod** [comment]
https://github.com/user-attachments/assets/cc75c43c-74f4-4d95-925f-128e62348305

@AtishayMsft FYI

**Comment by DrunkSunGod** [comment]
@AtishayMsft This can be repro-ed by  creating a bar chart and then naming the labels as

**Comment by AtishayMsft** [comment]
Could you add a change file for this fix. https://github.com/microsoft/fluentui/blob/master/docs/react-v9/contributing/dev-workflow.md#making-a-pull-request

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/react-charting/src/utilities/utilities.ts [hunk 1]
```diff
@@ -817,7 +817,7 @@ export function tooltipOfXAxislabels(xAxistooltipProps: any) {
       .on('mouseover', (event: any, d) => {
```
Comment (fabricteam) [inline packages/react-charting/src/utilities/utilities.ts:817]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33002&commitId=4bedfddd4a1ade429d4f35b1b381b630f5eeff4b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Bottom auto edge.chromium.png|2309| Changed|
|Callout.Top center.chromium.png|2178| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by DrunkSunGod** [comment]
@AtishayMsft Done. Can you please also let me know if I should create an issue and link it here (I have already filed a MSRC Vulnerability)

**Comment by AtishayMsft** [comment]
> @AtishayMsft Added the change file and also found this same thing in the Y axis labels so went ahead and changed that too. Can you please also let me know if I should create an issue and link it here (I have already filed a MSRC Vulnerability)

No, this is sufficient. Could you accept the CLA by replying to the comment above.

**Comment by DrunkSunGod** [comment]
> @DrunkSunGod please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by DrunkSunGod** [comment]
@AtishayMsft Please let me know if anything else needs to be done on my end. Thanks

**Review by AtishayMsft — APPROVED** [review]

**Comment by AtishayMsft** [comment]
> @AtishayMsft Please let me know if anything else needs to be done on my end. Thanks

Thanks for identifying the issue and fixing it. Merging this PR.

File: packages/react-charting/src/utilities/utilities.ts (hunk 1 above)
Comment (AtishayMsft) [reply packages/react-charting/src/utilities/utilities.ts:817]: changes in VR screenshots are unrelated to this fix.

**Pull request merged**